upgrading/topics/migrate_db.adoc

```diff
@@ -32,6 +32,33 @@ Or run this CLI command:
 When you start the server with this setting your database is automatically migrated if the database schema has changed
 in the new version.
 
+Creating an index on huge tables with millions of records can easily take a huge amount of time
+and potentially change major service disruption on upgrades.
+For those cases, we added the threshold (the number of records) for automated index creation.
```

Comment:
```suggestion
For those cases, we added a threshold (the number of records) for automated index creation.
By default, this threshold is `100000` records.
```

Reply:
I believe the "By default" part of this comment has not been addressed yet

Reply:
Thanks. I added this.